packages/server/src/sdk/app/automations/utils.ts

```diff
@@ -52,7 +52,7 @@ export async function getBuilderData(
     const rowActionName = await getRowActionName(tableId, rowActionId)
 
     result[automation._id!] = {
-      displayName: `${tableName}: ${automation.name}`,
+      displayName: rowActionName,
```

Comment:
Only backend change is removing the table name prefix from row action automation display names. This is no longer needed as we now group them by datasource.